sourcegraph/about
# Case study/ Hashicorp

Closes #5682 by making new HashiCorp case study page

### Changelog
- Adds HashiCorp case study page
- Renames `AuthorBio` => `StaffSpotlight` (no instances are actual authors, rather staff spotlighted and interviewed in the case study)
- Fixes broken UI for floating box on `UseChallengeSolutionResults` component from TW conversion

### Test
1. Ensure prettier has standardized the proposed changes.
2. Navigate to `/case-studies/hashicorp-uses-sourcegraph-to-streamline-cross-repository-code-search`
3. Test responsiveness across screen sizes
